### PR TITLE
fix: log results of make build

### PR DESCRIPTION
**Pull request by calabrla**
*Issue #, if available:*
N/A

*Description of changes:*
The Make builder does not log anything to stdout when it executes the make command. This change logs the output of the make command so developers can see the results of the execution. This follows the same pattern that exists in the `dotnetcli` builder.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws_lambda_builders/workflows/custom_make/make.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def run(self, args, env=None, cwd=None):
 
         out, err = p.communicate()
 
+        LOG.info(out.decode("utf8").strip())
```
Comment (mndeveci) [inline aws_lambda_builders/workflows/custom_make/make.py:87]: Should we also think about printing `err` output as well?

**Review by mndeveci — COMMENTED** [review]
Thanks for the PR @calabrla! Added single comment there

File: aws_lambda_builders/workflows/custom_make/make.py (hunk 1 above)
Comment (calabrla) [reply aws_lambda_builders/workflows/custom_make/make.py:87]: Isn't it sufficient that the `err` is raised by the `MakeExecutionError` if the return code is not `0` in lines 89-90?

**Review by mildaniel — APPROVED** [review]

**Comment by mildaniel** [comment]
nit: We normally log the outputs here at DEBUG level, where there are some exceptions like dotnet. I think for Makefile it makes sense to have as INFO due to the nature of it potentially being different for each customer's use case. Can we just leave a small comment noting why this is logged at INFO level to make it clear that this is the exception and not the rule.

**Comment by calabrla** [comment]
@mildaniel I've added a comment explaining why we're logging to INFO.

**Review by mndeveci — APPROVED** [review]

**Pull request merged**